app/views/mobile_phone/verify_create.html.haml

```diff
@@ -26,6 +26,11 @@
             oninput: "setCustomValidity('')"
         = button_tag "Verify"
 
+      - if flash[:errors]
```

Comment:
I think we need to check what the error is here.  We can't assume that it's always a "wrong token" error.  It might be "token expired", or something else.